rmw_opensplice_cpp/src/rmw_get_topic_info.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "rmw/error_handling.h"
+#include "rmw/rmw.h"
+#include "rmw/types.h"
+
+#include "identifier.hpp"
+#include "types.hpp"
+
+extern "C"
+{
+rmw_ret_t
+rmw_get_publishers_info_by_topic(
+  const rmw_node_t * node,
+  rcutils_allocator_t * allocator,
+  const char * topic_name,
+  bool no_mangle,
+  rmw_topic_info_array_t * publishers_info)
+{
+  (void) node;
+  (void) allocator;
+  (void) topic_name;
+  (void) no_mangle;
+  (void) publishers_info;
```

Comment:
Why is this required?

Reply:
To avoid unused warnings when building the package.